static/app/components/sidebar/sidebarItem.tsx

```diff
@@ -68,6 +70,8 @@ type Props = WithRouterProps & {
    */
   isNewSeenKeySuffix?: string;
   onClick?: (id: string, e: React.MouseEvent<HTMLAnchorElement>) => void;
+  organization?: Organization;
```

Comment:
@Neo-Zhixing I'd add a comment saying we only use this for analytics

Reply:
Comment added